### ISSUE TITLE
Data - Explore doesn't work 

### ISSUE THREAD

**Issue by Marie-Ann-S** [radiant-rstats/radiant#38]
Hi,

I use the radiant GUI in my stats course. 

After a new installation of the package (due to software updates of the university) I have a specific problem: The results for Data - Explore are not shown anymore in the GUI. The code for the explore function is provided for the report, but the results in the GUI just stay blank.

Do you have recommondations to fix this problem?

Best, 
Marie-Ann

**Comment by vnijs** [comment]
Hi Marie-Ann,

Glad to hear you are using Radiant for class! I'm guessing you installed Radiant from CRAN. A new version of the DT package that Radiant uses just came out and this is likely the issue. I recommend you use the following to install the latest version of Radiant and the packages it depends on:

`install.packages("radiant", repos = "https://radiant-rstats.github.io/minicran/", type = "binary")`

or use:

source("https://raw.githubusercontent.com/radiant-rstats/minicran/gh-pages/update.R")

For the latest documentation please see:

https://radiant-rstats.github.io/docs/index.html 

If that does not address your issue, I'll need more information to help figure out what is going on on your system. 

* Are you seeing the table in the Data > View tab?
* What about the Data > Pivot tab?
* Note that you now have to press the "Create table" button to generate results. 
* What OS and R version are you using?
* Are you running on a server or on laptops/desktops?
* Please provide results from `sessionInfo()`
* Any messages in the console or log files?
* Screenshots might be helpful as well.

![image](https://user-images.githubusercontent.com/2998956/35400932-0142a704-01ad-11e8-9496-e02ee6e13958.png)

Best,

Vincent

**Comment by Marie-Ann-S** [comment]
Thank you Vincent!

Yes, I installed radiant from CRAN and the installation of the DT package appeared different.  I will try out you alternative ways for installing Radiant as soon as possible, but I am pretty sure they will fix the problem. Thank you so much for the fast and helpful answer.

Best Marie-Ann

P.S. My students, who are the first cohort whom we forced to use R, are really grateful that Radiant exists.
